testbed/tests/trace_test.go

```diff
@@ -230,10 +285,10 @@ func TestTraceBallast1kSPSAddAttrs(t *testing.T) {
 	)
 }
 
-// verifySingleSpan sends a single span to Collector, waits until the span is forwarded
+// verifySingleJaegerSpan sends a single span to Collector, waits until the span is forwarded
 // and received by MockBackend and calls user-supplied verification function on
 // received span.
-func verifySingleSpan(
+func verifySingleJaegerSpan(
```

Comment:
This is a generic function that is independent from the format. It is not Jaeger-specific. The purpose of changing the name of the function is not clear.